docs/diagrams/choonwei/ListStudentSequenceDiagram.puml

```diff
@@ -0,0 +1,55 @@
+@startuml
+!include style.puml
```

Comment:
Actually you could include ../style.puml instead, so you don't have to make a new copy of the style.puml in your folder ah.

Reply:
Hahaha, yes I am aware, just thought that I might need to change it next time so might as well copy first. Will remove later on if it is not needed!